apps/api/src/campaign/campaign.controller.ts

```diff
@@ -25,8 +26,12 @@ export class CampaignController {
   }
 
   @Post('create-campaign')
-  async create(@Body() createDto: CreateCampaignDto) {
-    return await this.campaignService.createCampaign(createDto)
+  async create(
+    @Body() createDto: CreateCampaignDto,
+    @AuthenticatedUser() user: KeycloakTokenParsed,
+  ) {
+    const personId = !isAdmin(user) ? (user.sub as string) : undefined
```

Comment:
Here the `personId` differs from `user.sub` 

The `sub` prop equals to `person.keycloakId`